libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -500,7 +500,7 @@ void RangeFinder::detect_instance(uint8_t instance)
     if (type == RangeFinder_TYPE_LWI2C) {
         if (_address[instance]) {
             _add_backend(AP_RangeFinder_LightWareI2C::detect(*this, instance, state[instance],
-                hal.i2c_mgr->get_device(0, _address[instance])));
+                hal.i2c_mgr->get_device(1, _address[instance])));
```

Comment:
humn... this will actually break some boards. Correct way to fix this would be to add the define to HAL_Boards.h